sig-observability/concepts-and-designs/multi-tenant notification.md

```diff
@@ -62,16 +62,16 @@
 
 ## Requirements ##
 
-- User can define the channels for receiving notifications, including email, wechat, slack, webhook and dingtalk, by creating the corresponding receiver and config.
-- A user only has one receiver or config in the same type.
-- A user can update, delete, get the receiver, and config created by oneself.
-- A user can get the default config.
-- The receiver created by the user can use the config create by the user and the default config.
+- Every user can customize the channels for receiving notifications, including email, wechat, slack, webhook and dingtalk, by customize his own receiver and config. A user only has one receiver or config in the same type.
+
+![](../images/tenant-receiver-config.png)
+
+- The platform administrator can customize the global channel for receiving all notifications by customize the global receivers and the default configs. Each type of global receiver and default config can be only one.
```

Comment:
The platform administrator can customize the global channel for receiving all notifications by customizing the global receivers and the default configs. Only one global receiver or default config is allowed for each type.